public/app/plugins/datasource/loki/querybuilder/components/QueryPatternsModal.tsx

```diff
@@ -23,6 +23,27 @@ type Props = {
   onAddQuery?: (query: LokiQuery) => void;
 };
 
+const keepOperations: string[] = [
```

Comment:
Could we reuse some of the logic that already exists, because whenever we add a new operation, this would be an another place that we would need to update.

I was thinking about using `category` from `QueryBuilderOperationDefinition` which defines where are shown operations:
 
<img width="550" alt="image" src="https://github.com/grafana/grafana/assets/30407135/7ea601ae-5b19-415a-9a15-4d19f5d357a3">

WDYT?

Reply:
I think this is a very good idea! I'll see if we can use that.

Reply:
Using category now.